### ISSUE TITLE
use pnames for ontologies, owl:import, rdfs:isDefinedBy

### ISSUE THREAD

**Issue by VladimirAlexiev** [edmcouncil/fibo#992]
Currently:
```ttl
<https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Values/>
	a owl:Ontology ;
	owl:imports <https://spec.edmcouncil.org/fibo/ontology/master/latest/FND/Utilities/AnnotationVocabulary/> .

fibo-fnd-utl-val:hasDataValue
	rdfs:isDefinedBy <https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Values/> ;
```

I think this will read much nicer as:
```ttl
fibo-fnd-utl-val: a owl:Ontology ;
	owl:imports fibo-fnd-utl-av: .

fibo-fnd-utl-val:hasDataValue
	rdfs:isDefinedBy fibo-fnd-utl-val: ;
```

It will also emphasize that you have defined all prefixes.

**Comment by rivettp** [comment]
I agree, though this is to some extent an issue with the ontology-publisher since it's that which creates the Turtle serialization and adds (most of) the rdfs:isDefinedBy triples.
That said the original RDF/XML does include the full URIs for the Ontology and the imports - not sure why that is. I assume those too could be abbreviated by the ontology-publisher.

**Comment by VladimirAlexiev** [comment]
#995 suggests to use shortened URLs also in `seeAlso|dct:hasPart|versionIRI`.

About the last one: it is of course ok to NOT have a prefix for a historic versionIRI. But my observation is that at present FIBO uses only current versionIRIs.
